### DISCUSSION TITLE
광고 차단 불가

### DISCUSSION THREAD

**Discussion by xczsonn** [List-KR/community#70] [Ad-Shield]
https://klauncher.kr/ 해당 사이트에 접속하면
https://klauncher.kr/?adblock_detected=1 로 리디렉션 되고 사이트 이용이 불가합니다.

**Comment by piquark6046** [comment]
AdGuard DNS 필터를 꺼보세요.

**Comment by xczsonn** [comment]
애드가드 엣지 확장프로그램 사용 중인데 확장 프로그램 버전에도 dns 필터 옵션이 있나요
